From a92e14f32943f463a9cfbffe23f7153e6ed2b194 Mon Sep 17 00:00:00 2001
From: Scott K Logan <logans@cottsay.net>
Date: Wed, 20 Jan 2021 12:08:26 -0800
Subject: [PATCH] Set CMake flags for Fast-RTPS

Signed-off-by: Scott K Logan <logans@cottsay.net>
---
 rpm/template.spec.em | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/rpm/template.spec.em b/rpm/template.spec.em
index c0d00ea..24a841b 100644
--- a/rpm/template.spec.em
+++ b/rpm/template.spec.em
@@ -46,6 +46,8 @@ mkdir -p obj-%{_target_platform} && cd obj-%{_target_platform}
     -DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" \
     -DCMAKE_PREFIX_PATH="@(InstallationPrefix)" \
     -DSETUPTOOLS_DEB_LAYOUT=OFF \
+    -DINSTALL_EXAMPLES=OFF \
+    -DSECURITY=ON \
     ..
 
 %make_build
-- 
2.29.2

