From bf3250c75851e578720b828e4100d57d1e357c3f Mon Sep 17 00:00:00 2001
From: Scott K Logan <logans@cottsay.net>
Date: Wed, 20 Jan 2021 12:08:26 -0800
Subject: [PATCH] Set CMake flags for Fast-RTPS

Signed-off-by: Scott K Logan <logans@cottsay.net>
---
 rpm/template.spec.em | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/rpm/template.spec.em b/rpm/template.spec.em
index de5e5a79..b31a6b33 100644
--- a/rpm/template.spec.em
+++ b/rpm/template.spec.em
@@ -45,6 +45,8 @@ mkdir -p obj-%{_target_platform} && cd obj-%{_target_platform}
     -DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" \
     -DCMAKE_PREFIX_PATH="@(InstallationPrefix)" \
     -DSETUPTOOLS_DEB_LAYOUT=OFF \
+    -DINSTALL_EXAMPLES=OFF \
+    -DSECURITY=ON \
     ..
 
 %make_build
-- 
2.31.1

