From 34288986c682d57d3f9d85ae2a34b2ddaecad95c Mon Sep 17 00:00:00 2001
From: Chris Lalancette <clalancette@openrobotics.org>
Date: Thu, 25 Feb 2021 15:09:35 -0500
Subject: [PATCH] Make sure to set SECURITY=ON when building Fast-RTPS. (#12)

Signed-off-by: Chris Lalancette <clalancette@openrobotics.org>
---
 debian/rules.em | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 284a0714..ef66ba8e 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -37,7 +37,9 @@ override_dh_auto_configure:
 	dh_auto_configure -- \
 		-DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" \
 		-DCMAKE_PREFIX_PATH="@(InstallationPrefix)" \
-		$(BUILD_TESTING_ARG)
+		$(BUILD_TESTING_ARG) \
+		-DINSTALL_EXAMPLES=OFF \
+		-DSECURITY=ON
 
 override_dh_auto_build:
 	# In case we're installing to a non-standard location, look for a setup.sh
-- 
2.34.1

