From 1318307bcf6ada532d9b3c8f45f344afa383c61e Mon Sep 17 00:00:00 2001
From: Chris Lalancette <clalancette@openrobotics.org>
Date: Fri, 26 Feb 2021 11:00:51 -0500
Subject: [PATCH] Reenable SECURITY for Dashing Fast-RTPS. (#13)

Signed-off-by: Chris Lalancette <clalancette@openrobotics.org>
---
 debian/rules.em | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 3342408..d56bb29 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -31,7 +31,9 @@ override_dh_auto_configure:
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
 	dh_auto_configure -- \
 		-DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" \
-		-DCMAKE_PREFIX_PATH="@(InstallationPrefix)"
+		-DCMAKE_PREFIX_PATH="@(InstallationPrefix)" \
+		-DINSTALL_EXAMPLES=OFF \
+		-DSECURITY=ON
 
 override_dh_auto_build:
 	# In case we're installing to a non-standard location, look for a setup.sh
-- 
2.25.1

