From 58093d2aa8796652516c82352f08f106b9ceff16 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Steven!=20Ragnar=C3=B6k?= <steven@nuclearsandwich.com>
Date: Thu, 15 Nov 2018 19:35:38 -0800
Subject: [PATCH] Set CMake flags for Fast-RTPS.

---
 debian/rules.em | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 33424088..d56bb299 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -31,7 +31,9 @@ override_dh_auto_configure:
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
 	dh_auto_configure -- \
 		-DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" \
-		-DCMAKE_PREFIX_PATH="@(InstallationPrefix)"
+		-DCMAKE_PREFIX_PATH="@(InstallationPrefix)" \
+		-DINSTALL_EXAMPLES=OFF \
+		-DSECURITY=ON
 
 override_dh_auto_build:
 	# In case we're installing to a non-standard location, look for a setup.sh
-- 
2.30.2

