From 91e5a02bde032dae356538b6ee9bc8c2f1bd8fbf Mon Sep 17 00:00:00 2001
From: Shane Loretz <shane.loretz@gmail.com>
Date: Mon, 21 Mar 2022 14:16:17 -0700
Subject: [PATCH 2/2] Set APPEND_PROJECT_NAME_TO_INCLUDEDIR to ON

---
 debian/rules.em | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index ef66ba8e..021135b6 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -39,7 +39,8 @@ override_dh_auto_configure:
 		-DCMAKE_PREFIX_PATH="@(InstallationPrefix)" \
 		$(BUILD_TESTING_ARG) \
 		-DINSTALL_EXAMPLES=OFF \
-		-DSECURITY=ON
+		-DSECURITY=ON \
+		-DAPPEND_PROJECT_NAME_TO_INCLUDEDIR=ON
 
 override_dh_auto_build:
 	# In case we're installing to a non-standard location, look for a setup.sh
-- 
2.34.1

