From 79d44dd5dcaf30c3d83612432849c0affd4560a7 Mon Sep 17 00:00:00 2001
From: Mikael Arguedas <mikael@osrfoundation.org>
Date: Mon, 16 Jul 2018 14:07:11 -0700
Subject: [PATCH 4/4] dont install examples using -DINSTALL_EXAMPLES=OFF

---
 debian/rules.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index a56087e..32d5dd3 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -26,7 +26,7 @@ override_dh_auto_configure:
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
-	dh_auto_configure -- -DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" -DSECURITY=ON
+	dh_auto_configure -- -DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" -DINSTALL_EXAMPLES=OFF -DSECURITY=ON
 
 override_dh_auto_build:
 	# In case we're installing to a non-standard location, look for a setup.sh
-- 
2.7.4

